### PR TITLE
Update project instructions and submission instructions

### PR DESCRIPTION
**Pull request by ry-oc**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: content/sql/shop-project/_index.md [hunk 1]
```diff
@@ -74,31 +74,33 @@ Save all of your instructions in script files - you will submit these files on G
...
 ### Orders Table
 
-| OrderId (int) | ProductID (int) | PaymentID (int) | FulfilledByEmployeeID (int) | DateRequired (datetime) | DateShipped (datetime) | Status (varchar20) |
+| Id (int) | ProductID (int) | PaymentID (int) | FulfilledByEmployeeID (int) | DateRequired (datetime) | DateShipped (datetime) | Status (varchar20) |
 | ------------- | --------------- | --------------- | --------------------------- | ----------------------- | ---------------------- | ------------------ |
 | 1             | 1               | 1               | 2                           | 05-09-2018              |                        | Not shipped        |
 | 2             | 1               | 2               | 2                           | 04-09-2018              | 03-09-2018             | Shipped            |
 | 3             | 3               | 3               | 3                           | 06-09-2018              |                        | Not shipped        |
 
+Note: When creating tables you will note that each table has an ID column, when joining data between tables the foreign key must include the name of the table the data came from, i.e. `CustomerID` in the above table.
```
Comment (Andy-Nkumane) [inline content/sql/shop-project/_index.md]:
```suggestion
Note: When creating tables you will note that each table has an ID column, when joining data between tables the foreign key must include the name of the table the data came from, i.e. `ProductID` in the above table.
```

File: content/sql/shop-project/_index.md [hunk 2]
```diff
@@ -74,31 +74,33 @@ Save all of your instructions in script files - you will submit these files on G
...
+| ID (int) | FirstName (varchar50) | LastName (varchar50) | Email (varchar100) | JobTitle (varchar20) |
 | ---------------- | --------------------- | -------------------- | ------------------ | -------------------- |
 | 1                | Kani                  | Matthew              | mat@gmail.com      | Manager              |
 | 2                | Lesly                 | Cronje               | LesC@gmail.com     | Clerk                |
 | 3                | Gideon                | Maduku               | m@gmail.com        | Accountant           |
 
 ### Orders Table
 
-| OrderId (int) | ProductID (int) | PaymentID (int) | FulfilledByEmployeeID (int) | DateRequired (datetime) | DateShipped (datetime) | Status (varchar20) |
+| Id (int) | ProductID (int) | PaymentID (int) | FulfilledByEmployeeID (int) | DateRequired (datetime) | DateShipped (datetime) | Status (varchar20) |
```
Comment (Andy-Nkumane) [inline content/sql/shop-project/_index.md]:
```suggestion
| ID (int) | ProductID (int) | PaymentID (int) | FulfilledByEmployeeID (int) | DateRequired (datetime) | DateShipped (datetime) | Status (varchar20) |
```

File: content/sql/shop-project/_index.md [hunk 3]
```diff
@@ -74,31 +74,33 @@ Save all of your instructions in script files - you will submit these files on G
...
 | ------------- | --------------- | --------------- | --------------------------- | ----------------------- | ---------------------- | ------------------ |
 | 1             | 1               | 1               | 2                           | 05-09-2018              |                        | Not shipped        |
 | 2             | 1               | 2               | 2                           | 04-09-2018              | 03-09-2018             | Shipped            |
 | 3             | 3               | 3               | 3                           | 06-09-2018              |                        | Not shipped        |
 
+Note: When creating tables you will note that each table has an ID column, when joining data between tables the foreign key must include the name of the table the data came from, i.e. `CustomerID` in the above table.
 ### Payments Table
 
-| CustomerId (int) | PaymentID (int) | PaymentDate (datetime) | Amount (decimal) |
+| CustomerId (int) | ID (int) | PaymentDate (datetime) | Amount (decimal) |
```
Comment (Andy-Nkumane) [inline content/sql/shop-project/_index.md]:
```suggestion
| CustomerID (int) | ID (int) | PaymentDate (datetime) | Amount (decimal) |
```

File: content/sql/shop-project/_index.md [hunk 4]
```diff
@@ -74,31 +74,33 @@ Save all of your instructions in script files - you will submit these files on G
...
 | ---------------- | --------------- | ---------------------- | ---------------- |
 | 1                | 1               | 01-09-2018             | R150.75          |
 | 5                | 2               | 03-09-2018             | R150.75          |
 | 4                | 3               | 03-09-2018             | R700.60          |
 
+Note: When creating tables you will note that each table has an ID column, when joining data between tables the foreign key must include the name of the table the data came from, i.e. `CustomerID` in the above table.
 ### Products Table
 
-| ProductId (int) | ProductName (varchar100) | Description (varchar300)                                                    | BuyPrice (decimal) |
+| Id (int) | ProductName (varchar100) | Description (varchar300)                                                    | BuyPrice (decimal) |
```
Comment (Andy-Nkumane) [inline content/sql/shop-project/_index.md]:
```suggestion
| ID (int) | ProductName (varchar100) | Description (varchar300)                                                    | BuyPrice (decimal) |
```

File: content/sql/shop-project/_index.md (hunk 1 above)
Comment (Andy-Nkumane) [reply content/sql/shop-project/_index.md]: to avoid possible confusion, it is best to be accurate as possible

**Review by Andy-Nkumane — CHANGES_REQUESTED** [review]
getting there:
* there is some inconsistency in the table id's

File: content/sql/shop-project/_index.md [hunk 5]
```diff
@@ -74,31 +74,33 @@ Save all of your instructions in script files - you will submit these files on G
...
 | ------------- | --------------- | --------------- | --------------------------- | ----------------------- | ---------------------- | ------------------ |
 | 1             | 1               | 1               | 2                           | 05-09-2018              |                        | Not shipped        |
 | 2             | 1               | 2               | 2                           | 04-09-2018              | 03-09-2018             | Shipped            |
 | 3             | 3               | 3               | 3                           | 06-09-2018              |                        | Not shipped        |
 
+Note: When creating tables you will note that each table has an ID column, when joining data between tables the foreign key must include the name of the table the data came from, i.e. `ProductID` in the above table.
 ### Payments Table
 
-| CustomerId (int) | PaymentID (int) | PaymentDate (datetime) | Amount (decimal) |
+| CustomerID (int) | ID (int) | PaymentDate (datetime) | Amount (decimal) |
```
Comment (Samantha-Hampton) [inline content/sql/shop-project/_index.md]: tiny thing but just for consistency across the tables should `ID (int)` not be the first column of the payments table?

**Review by Samantha-Hampton — CHANGES_REQUESTED** [review]

File: content/sql/shop-project/_index.md (hunk 5 above)
Comment (ry-oc) [reply content/sql/shop-project/_index.md]: good point, will change it now

**Review by Samantha-Hampton — APPROVED** [review]

**Review by Andy-Nkumane — APPROVED** [review]

**Pull request merged**